ports/fastcdr/portfile.cmake

```
@@ -13,21 +13,17 @@ vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
     PREFER_NINJA
     OPTIONS
-        -DLICENSE_INSTALL_DIR=share/fastcdr
         -DCMAKE_DISABLE_FIND_PACKAGE_GTest=ON
 )
 
 vcpkg_install_cmake()
 
-vcpkg_fixup_cmake_targets(CONFIG_PATH share/fastcdr/cmake)
-
-file(READ "${CURRENT_PACKAGES_DIR}/share/fastcdr/fastcdr-config.cmake" _contents)
-string(REPLACE "include(\${fastcdr_LIB_DIR}/fastcdr/cmake/fastcdr-targets.cmake)" "include(\${CMAKE_CURRENT_LIST_DIR}/fastcdr-targets.cmake)" _contents "${_contents}")
-file(WRITE "${CURRENT_PACKAGES_DIR}/share/fastcdr/fastcdr-config.cmake" "${_contents}")
+vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/fastcdr TARGET_PATH share/fastcdr)

```

```suggestion
vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/fastcdr)
```
`share/${PORT}` is the default for `TARGET_PATH`.

Please test that the config file still works in a test CMake project:
```cmake
find_package(fastcdr CONFIG REQUIRED)
```